presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +may cause better responsiveness for new task but       +
++                                          +                                          +                                    +possibly decreases CPU utilization. Higher value makes +
++                                          +                                          +                                    +context switching faster with the cost of additional   +
++                                          +                                          +                                    +memory. The general rules of managing drivers is that  +
++                                          +                                          +                                    +if there is possibility of assigning a split to driver +
++                                          +                                          +                                    +it is assigned if: there are less then ``3`` drivers   +
++                                          +                                          +                                    +assigned to given task OR there is less drivers on     +
++                                          +                                          +                                    +worker then ``task.min-drivers`` OR the task has been  +
++                                          +                                          +                                    +enqueued with ``force start`` property.                +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++  ``task.operator-pre-allocated-memory``  +          ``String`` (data size)          +             ``16 MB``              +Memory preallocated for each driver in query           +
++                                          +                                          +                                    +execution. Increasing this value may cause less        +
++                                          +                                          +                                    +efficient memory usage but allows to fail fast in low  +
++                                          +                                          +                                    +memory environment more frequently.                    +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.share-index-loading``       +               ``Boolean``                +             ``false``              +It allows to control whether index lookups join has    +
```

Comment:
Again, we cannot talk about index join tuning without more explanation what they are and how to get them.